### ISSUE TITLE
Set a standard for choosing upgrade plan names

### ISSUE THREAD

**Issue by ValarDragon** [cosmos/gaia#902]
I suggest we enforce a standard for all upgrade plan names to correspond to release version it is coming in. The name of an upgrade dictates the name of cosmovisor folder a validator should keep their new binary in. It would be much simpler if the name of an upgrade always matched the name of the git tag. 

The next hub upgrade has an upgrade name of `Gravity-DEX` (see https://github.com/cosmos/gaia/blob/main/app/app.go#L418), which adds mental overhead to all validators for determining the name of the upgrade. (I had to cross-check the code and gov proposal to ensure I got the name right when setting up the cosmovisor folders)

I suggest for simplicity that in the future the upgrade plan names just be called `v6`, or `vX` in general.

If using just `vX` is unpalatable, lets at least make casing simpler.

____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by nooomski** [comment]
Bot closed this by accident. Opening again. @okwme can we discuss this on the next internal call?

**Comment by okwme** [comment]
We discussed on the hub team and are in favor of a combined approach.
We've started using:
 * v7-Theta
 * v8-Rho
What do you think @ValarDragon ?

**Comment by ValarDragon** [comment]
my ranking of names personally is:
```
v7
v7-theta
v7-Theta
```

But not really a strong preference. It not being the name of a feature is a huge win! I think I'd personally cross-check against the code to make sure I got the naming right, which is fine

**Comment by yaruwangway** [comment]
Hi as discussed, we use `v07` rather than `7` in the upgrade name still?

**Comment by ValarDragon** [comment]
Another alternative is to in a future release, make cosmovisor case & punctuation insensitive

**Comment by okwme** [comment]
we rediscussed the `v0x-Name` versus `v7-Name` and find `v7-Name` preferrable.
Not opposed to `vx-name` but based on the fact these are names don't mind keeping them capitalized.
We did `Vega` with capital last time so also would be consistent with that one.
So fro the records the next upgrades with be:
```
v7-Theta
v8-Rho
v9-Lambda
```
Thanks for bringing this up @ValarDragon think it's a big improvement 🎉
